doc/api/api.md

```diff
@@ -17,6 +17,9 @@ To learn about what specific calls can be made read the [wallet foreign API doc]
 
 ### Wallet Owner API
 
-The wallet owner API is an endpoint to manage the user wallet: broadcast transaction, sign transaction, see the current balance... This REST API can be started with the `grin wallet owner_api` command and will listen on `localhost:3420`. This endpoint must **never** be exposed to the outside world.
+The wallet owner API is an endpoint to manage the user wallet: broadcast transaction, sign transaction, see the current balance... This REST API can be started with the `grin wallet owner_api` command and will listen on `localhost:13420` for --floonet or `localhost:3420` for --mainnet.
```

Comment:
Thank you, fixed

Reply:
And the reason for keeping the distinction here is because there might be people using almost-updated mainnet code releases that are using 13420 instead of 3420 as the default.

Do you think it's best to add a comment or "errata" about this also in the documentation? Something like: "If your grin owner_api is using the 13420 port but is on mainnet, then you're using an outdated version of grin. Please upgrade to get this and other bugfixes"